components/input/styles.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components'
+import COLORS from 'shared/colors'
+import MEASURES from 'shared/measures'
+
+const Input = styled.input`
+  -webkit-appearance: none;
```

Comment:
I think styled-components does autoprefixer by default (because it probably uses React's CSS injection)